dev-docs/modules/userid-submodules/intentiq.md

```diff
@@ -31,15 +31,16 @@ Please find below list of parameters that could be used in configuring Intent IQ
 
 {: .table .table-bordered .table-striped }
 
-| Param under userSync.userIds[] | Scope    | Type   | Description                                                                                                                                                                                                  | Example         |
-| ------------------------------ | -------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------- |
-| name                           | Required | String | The name of this module: "intentIqId"                                                                                                                                                                        | `"intentIqId"`  |
-| params                         | Required | Object | Details for IntentIqId initialization.                                                                                                                                                                       |                 |
-| params.partner                 | Required | Number | This is the partner ID value obtained from registering with IntentIQ.                                                                                                                                        | `1177538`       |
-| params.percentage              | Required | Number | This a percentage value for our A/B testing group distribution. The values supposed to be in range of 0 to 100. We suggest to set it to 95 percent for optimal balance ofbetween prefromance and preceision. | `95`            |
```

Comment:
doesn't removing this and adding callback break implementations?

Reply:
No, it doesn't break the implementation. We now receive the percentage and A/B testing group from our server. Additionally, a callback is added to synchronize partners with us, as they typically have their own logic implemented on the page based on the callback.